test/__mocks__/data/sources/shopify/response.json

```diff
@@ -0,0 +1,253 @@
+{
+  "shopify_test1": {
+    "anonymousId": "anon_shopify_test1",
+    "cart": {
+      "line_items": [
+        {
+          "id": 44323300999443,
+          "properties": null,
+          "quantity": 1,
+          "variant_id": 44323300999443,
+          "key": "44323300999443:70b21df78d403673a8b5dbafd62f0559",
+          "discounted_price": "30.00",
+          "discounts": [],
+          "gift_card": false,
+          "grams": 0,
+          "line_price": "30.00",
+          "original_line_price": "30.00",
+          "original_price": "30.00",
+          "price": "30.00",
+          "product_id": 8100575674643,
+          "sku": "",
+          "taxable": true,
+          "title": "Shirt 2 - LARGE",
+          "total_discount": "0.00",
+          "vendor": "testAnant",
+          "discounted_price_set": {
+            "shop_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            },
+            "presentment_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            }
+          },
+          "line_price_set": {
+            "shop_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            },
+            "presentment_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            }
+          },
+          "original_line_price_set": {
+            "shop_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            },
+            "presentment_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            }
+          },
+          "price_set": {
+            "shop_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            },
+            "presentment_money": {
+              "amount": "30.0",
+              "currency_code": "GBP"
+            }
+          },
+          "total_discount_set": {
+            "shop_money": {
+              "amount": "0.0",
+              "currency_code": "GBP"
+            },
+            "presentment_money": {
+              "amount": "0.0",
+              "currency_code": "GBP"
+            }
+          }
+        }
+      ]
+    }
```

Comment:
We are not storing cart details anymore. We should update these responses accordingly.

Reply:
Done